utils/testingUtil.js

```diff
@@ -75,6 +80,45 @@ const performSignedGETRequest = (server, url, privateKey = null) => new Promise(
     });
 });
 
+const fakeTipsAndUpdateDB = dbsToClear => async (fakeData, clearData = true) => {
```

Comment:
rename this to something more suiting

Reply:
can you provide a PR

Reply:
https://github.com/aeternity/tipping-community-backend/pull/312